programs/zstdcli.c

```diff
@@ -1213,6 +1227,23 @@ int main(int const argCount, const char* argv[])
         if (adaptMin > cLevel) cLevel = adaptMin;
         if (adaptMax < cLevel) cLevel = adaptMax;
 
+        if (showDefaultCParams) {
+            size_t fileNb;
+            for (fileNb = 0; fileNb < (size_t)filenames->tableSize; fileNb++) {
+                const size_t fileSize = UTIL_isRegularFile(filenames->fileNames[fileNb]) ? (size_t)UTIL_getFileSize(filenames->fileNames[fileNb]) : 0;
+                const size_t dictSize = dictFileName != NULL ? (size_t)UTIL_getFileSize(dictFileName) : 0;
+                const ZSTD_compressionParameters cParams = ZSTD_getCParams(cLevel, (unsigned long long)fileSize, dictSize);
+                if (fileSize) DISPLAY("%s (%u bytes)\n", filenames->fileNames[fileNb], (unsigned)fileSize);
+                else DISPLAY("%s (src size unknown)\n", filenames->fileNames[fileNb]);
+                DISPLAY(" - windowLog    : %u\n", (unsigned)cParams.windowLog);
+                DISPLAY(" - chainLog     : %u\n", (unsigned)cParams.chainLog);
+                DISPLAY(" - searchLog    : %u\n", (unsigned)cParams.searchLog);
+                DISPLAY(" - minMatch     : %u\n", (unsigned)cParams.minMatch);
+                DISPLAY(" - targetLength : %u\n", (unsigned)cParams.targetLength);
```

Comment:
minor: all these values are supposed to be already `unsigned`, so there is no value in casting them again.